src/test/java/com/google/sps/TimingPropertiesTest.java

```diff
@@ -214,8 +236,7 @@ public void giveUserPoints_validParameters() {
         user.setProperty("score", 0);
         datastore.put(user);
  
-        Boolean actual = timing_properties_test.giveUserPoints(true, user, datastore);
+        boolean actual = timing_properties_test.giveUserQuizTakenPoints(true, user, datastore);
         Assert.assertEquals(true, actual);
     }
-
-}
\ No newline at end of file
+}
```

Comment:
We should add another test that makes sure that a user's score is actually increased. So one where we make a user entity (maybe with a non-zero and a zero score, so two test cases), then add 20 points, and ensure their score is score + 20.

Also, we should add a test case where the user never had a score property set (aka this line never exists in the test: user.setProperty("score", 0);)